src/applications/personalization/profile360/components/Vet360TransactionInlineErrorMessage.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 
-import {
-  hasUserIsDeceasedError
-} from '../util/transactions';
+// import {
```

Comment:
Just in case we need to add a specific error message again, the flow is already there in the comments.

Reply:
Could you condense this to a one-liner saying something to the effect of "we can add other error messages here by adding a condition and setting content"?

Typically we don't want to leave commented code like this because it create ambiguity in the code base. It is easy to re-add these lines if another error message is needed, especially if there is one-line explanatory comment is fine for people unfamiliar with the codebase

Reply:
Ah okay gotcha. I'll just remove the code there and clean up the file. There are already two other error messaging files that other devs can check for reference.